example/src/classes/view/main/main-view.js

```diff
@@ -0,0 +1,33 @@
+import ElementBuilder from '../../util/element/element-builder';
+import View from '../view';
+
+const CssClasses = {
+    MAIN: 'main',
+};
+
+export default class MainView extends View {
+    constructor() {
+        super();
+        this.htmlElement = this.createView();
+    }
+
+    /**
+     * В данном методе входящим параметром является любой объект унаследованный от класса View, т.к. для
+     * использования объекта требуется метод getHtmlElement() этого класса. Это демонстрирует один из принципов
+     * SOLID (буква L - Liskov Substitution Principle) - когда мы можем вместо объекта родительского класса
```

Comment:
This code have nothing to do with Liskov. It shows how inheritance works, not Liskov.